zerver/lib/webhooks/git.py

```diff
@@ -98,13 +120,15 @@ def get_pull_request_event_message(
         main_message += '\n' + CONTENT_MESSAGE_TEMPLATE.format(message=message)
     return main_message.rstrip()
 
+
```

Comment:
Please remove double empty lines between functions (our style is to have just one)